metagraph/src/graph/graph_extensions/node_rc.cpp

```diff
@@ -31,10 +32,19 @@ NodeRC::NodeRC(const DeBruijnGraph &graph, bool construct_index) : graph_(&graph
         return;
 
     const BOSS &boss = dbg_succ->get_boss();
+    const auto *mask = dbg_succ->get_mask();
+    std::unique_ptr<bit_vector_stat> computed_mask;
+    if (!mask) {
+        auto sdsl_mask = boss.mark_all_dummy_edges(get_num_threads());
+        sdsl_mask.flip();
+        computed_mask = std::make_unique<bit_vector_stat>(std::move(sdsl_mask));
```

Comment:
So bit_vector_stat computes rank and select support which are unnecessary here.
Better use sdsl::bit_vector

Reply:
do you think it's good now? The only unnecessary overhead is that it now computes the number of set bits

Reply:
but why not use the sdsl bitmap?

Reply:
never mind, it's good